src/core/tail.js

```
@@ -0,0 +1,104 @@
+        return { stdout: '', stderr: msg, env: Object.assign({}, env, { '?': 1 }) };
+      }
+    }
+  } else {
+    input = stdin;
+  }
+  // process content
+  let stdout = '';
+  if (mode === 'lines') {
+    const raw = countValue;
+    const sign = raw && (raw[0] === '+' || raw[0] === '-') ? raw[0] : null;
+    const num = raw !== undefined ? parseInt(raw.replace(/^[+-]/, ''), 10) : 10;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)